utils/setup-server.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -o errexit
+set -o nounset
+
+sed "\
```

Comment:
Rather than using sed, I would provide a specific settings_sqlite.py  and a settings_postgres.py . I'm okay doing this now, but in any case, I think we can improve how this is done:
https://github.com/FOSDEM/volunteers/issues/44